### DISCUSSION TITLE
How to find the blocked IPs in firewalld?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [firewalld/firewalld#1253] [Q&A]
I found when my centos firewalld in started status, the entrance direction IP access in high frequency, the IP will be block by firewalld.

but in the [official site](https://firewalld.org/), I don't find where is the place state of the function.

I think there should have the official notes, and in firewalld I could see where is the IPs it blocked.

**Comment by erig0** [comment]
Firewalld blocks by default. Unless you specifically allow something or it's part of the defaults, e.g. ssh, then the traffic will be blocked.

Please share more information about what behavior you expect.

**Reply by liaodalin19903** [reply]
I find if a IP access the Linux with low frequency, the firewalld will not block it, but if the IP with a hight frequency, the IP will be blocked, (I prove it by turn off the firewalld)

where is the official statement of this function of firewalld? 

and if we can check the blocked IPs by firewalld?

**Reply by erig0** [reply]
> where is the official statement of this function of firewalld?

Firewalld does not rate limit new connections by default. How are you testing this? ping? http? ssh?

Please share your firewalld configuration: `--list-all-zones` and `--list-all-policies`.

> and if we can check the blocked IPs by firewalld?

Firewalld is block by default. You should check for allowed things. If it's not explicitly allowed, the it's blocked. It does not make sense to enumerate "all blocked IPs".
